game.py

```diff
@@ -821,10 +827,15 @@ def save_game(self, auto: bool = False):
         Saves the current game with the current timestamp as the file name.
         """
         # Only maintain 3 autosaves at a time, delete the oldest if we already have 3 before saving the next.
-        if auto and len(autosaves := list(filter(lambda fn: fn.startswith("auto"), os.listdir("saves/")))) == 3:
+        if auto and len(
+                autosaves := list(filter(lambda fn: fn.startswith(AUTOSAVE_PREFIX), os.listdir(SAVES_DIR)))) == 3:
             autosaves.sort()
-            os.remove(f"saves/{autosaves[0]}")
-        save_name = f"saves/{'auto' if auto else ''}save-{datetime.datetime.now().isoformat(timespec='seconds')}.json"
+            os.remove(os.path.join(SAVES_DIR, {autosaves[0]}))
```

Comment:
Don't think the braces are needed here, they were previously required because it was in a format string.

Reply:
Yeah this actually crashes the game for me 💀

Reply:
Good call, I'm curious if it's actually that causing your crash or something else though. Despite the brackets it seemed to work on Win/Ubuntu. Will look into it though, maybe missed a commit removing these or something.

Reply:
I also don't think I tested the autosave remove part so that might explain it!

Reply:
Replicated it on my end now and pushed a fix. My bad!

Reply:
Fixed in [d1ad791](https://github.com/ChrisNeedham24/microcosm/pull/70/commits/d1ad79100f613f104d44f75c4c56ee66be9a933c)